osvr/RenderKit/RenderManagerOpenGLC.h

```diff
@@ -79,6 +81,18 @@ typedef struct OSVR_OpenGLContextParams {
     OSVR_CBool visible;
 } OSVR_OpenGLContextParams;
 
+typedef struct OSVR_RenderTimingInfo {
```

Comment:
[Question] Should this definition be moved to [`RenderManagerC.h` line 102](https://github.com/sensics/OSVR-RenderManager/blob/custom-gl-timing-info/osvr/RenderKit/RenderManagerC.h#L102) so it can be used by other renderers?